docs/shared/recipes/module-federation/dynamic-mfe-angular.md

```diff
@@ -36,46 +41,97 @@ Here's the source code of the final result for this guide.
 
 ### Create an Nx Workspace
 
-To start with, we need to create a new Nx Workspace. We can do this easily with:
+To start with, we need to create a new Nx Workspace and add the Nx Angular plugin. We can do this easily with:
 
 {% tabs %}
 {% tab label="npm" %}
 
+```{% command="npx create-nx-workspace@latest ng-mf" path="~/" %}
+ NX   Let's create a new workspace [https://nx.dev/getting-started/intro]
+
+✔ Which stack do you want to use? · none
+✔ Package-based monorepo, integrated monorepo, or standalone project? · integrated
+✔ Which CI provider would you like to use? · skip
+✔ Would you like remote caching to make your build faster? · skip
+
+ NX   Creating your v19.7.4 workspace.
+
+✔ Installing dependencies with npm
+✔ Successfully created the workspace: ng-mf.
+```
+
+Next run:
+
 ```shell
-npx create-nx-workspace ng-mf
+cd ng-mf
+npx nx add @nx/angular
 ```
 
 {% /tab %}
 {% tab label="yarn" %}
 
+```{% command="yarn create nx-workspace ng-mf" path="~/" %}
+ NX   Let's create a new workspace [https://nx.dev/getting-started/intro]
+
+✔ Which stack do you want to use? · none
+✔ Package-based monorepo, integrated monorepo, or standalone project? · integrated
+✔ Which CI provider would you like to use? · skip
+✔ Would you like remote caching to make your build faster? · skip
+
+ NX   Creating your v19.7.4 workspace.
+
+✔ Installing dependencies with yarn
+✔ Successfully created the workspace: ng-mf.
+```
+
+Next run:
+
 ```shell
-yarn create nx-workspace ng-mf
+cd ng-mf
+yarn nx add @nx/angular
 ```
 
 {% /tab %}
 {% tab label="pnpm" %}
 
+```{% command="pnpx create-nx-workspace@latest ng-mf" path="~/" %}
+ NX   Let's create a new workspace [https://nx.dev/getting-started/intro]
+
+✔ Which stack do you want to use? · none
+✔ Package-based monorepo, integrated monorepo, or standalone project? · integrated
+✔ Which CI provider would you like to use? · skip
+✔ Would you like remote caching to make your build faster? · skip
+
+ NX   Creating your v19.7.4 workspace.
+
+✔ Installing dependencies with pnpm
+✔ Successfully created the workspace: ng-mf.
+```
+
+Next run:
+
 ```shell
-pnpm create nx-workspace ng-mf
+cd ng-mf
+pnpx nx add @nx/angular
 ```
 
 {% /tab %}
 {% /tabs %}
 
-You'll be prompted a few questions. Pick the `Angular` stack, `Integrated Monorepo` layout and the `webpack` bundler. You can use the default values for the rest of the prompts. We won't use the application that gets generated by default on this guide, so you can remove it.
-
 ### Creating our applications
 
 We need to generate two applications that support Module Federation.
 
-We'll start with the **Admin Dashboard** application which will act as a host application for the Micro-Frontends (_MFEs_):
+We'll start with the **Admin Dashboard** application which will act as a host application for the Micro-Frontends (
+_MFEs_):
 
 ```shell
-nx g @nx/angular:host apps/dashboard --prefix=ng-mf
+nx g @nx/angular:host dashboard --directory=apps/dashboard --prefix=ng-mf
```

Comment:
This is fine, but maybe we shouldn't change it now, given that we'll soon swap the positional arg to be the `directory` instead of the `name`, and we'll need to update this again. Since the current information is also not incorrect, we could keep it as is until we need have to make that change.

Feel free to keep the change if you like, but if you decide to revert it, please do the same with the other similar changes in this PR.